docs/conf.py

```diff
@@ -13,75 +13,79 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('../neuromllite'))
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath("../neuromllite"))
+sys.path.insert(0, os.path.abspath(".."))
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'NeuroMLlite'
-copyright = '2021, NeuroML contributors'
-author = 'NeuroML contributors'
+project = "NeuroMLlite"
+copyright = "2021, NeuroML contributors"
+author = "NeuroML contributors"
 
 
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.todo',
-              'sphinxcontrib.bibtex']
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.doctest",
+    "sphinx.ext.todo",
+    "sphinxcontrib.bibtex",
+]
 
-bibtex_bibfiles = ['refs.bib']
+bibtex_bibfiles = ["refs.bib"]
 
 # Include TODOs in docs
 todo_include_todos = True
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # The suffix of source filenames.
-source_suffix = '.rst'
+source_suffix = ".rst"
 
 # The short X.Y version.
 version = ""
-for aline in open('../neuromllite/__init__.py'):
+for aline in open("../neuromllite/__init__.py"):
     # space here is important since __version__ is used in generation of
     # version_info also
-    if '__version__ =' in aline:
+    if "__version__ =" in aline:
         version = aline.split("'")[1]
```

Comment:
We need to update this manually once to `"\""` because black changes the single quote in `__init__.py` to a double quote.

Reply:
Thanks for spotting that. Updated.